releasenotes/notes/move-es-version-vars-17c22cfd558a68d5.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
OK, but why are those package lists variablised? Why not just have them be static names?